keyboards/handwired/philoandgunge/nyaaaah54_rp2040/info.json

```diff
@@ -0,0 +1,543 @@
+{
+  "manufacturer": "0xPHGU",
+  "keyboard_name": "nyaaaah54_rp2040",
+  "maintainer": "jkaunert",
+  "bootloader": "rp2040",
+  "debounce": 5,
+  "diode_direction": "COL2ROW",
+  "features": {
+    "bootmagic": true,
+    "extrakey": true,
+    "mousekey": true,
+    "nkro": true
+  },
+  "matrix_pins": {
+    "cols": [
+      "GP21",
+      "GP20",
+      "GP19",
+      "GP18",
+      "GP17",
+      "GP16",
+      "GP15",
+      "GP14",
+      "GP13",
+      "GP12",
+      "GP11",
+      "GP10"
+    ],
+    "rows": [
+      "GP5",
+      "GP6",
+      "GP7",
+      "GP8",
+      "GP9"
+    ]
+  },
+  "processor": "RP2040",
+  "qmk": {
+    "tap_keycode_delay": 10
+  },
+  "url": "https://philoandgunge.com",
+  "usb": {
+    "device_version": "1.0.0",
+    "pid": "0x0000",
+    "vid": "0xFEED"
+  },
+  "ws2812": {
+    "driver": "vendor",
+    "pin": "GP29"
+  },
+  "layouts": {
```

Comment:
The formatter does not format like this, please use the command previously mentioned.